.github/workflows/lint.yaml

```diff
@@ -4,7 +4,7 @@ on:
 
 jobs:
   cpp_lint:
-    name: Format and lint
+    name: C++ format and lint
```

Comment:
Why C++?

Reply:
I want to differentiate the name from the new Python format/lint.
The job name is cpp_lint. Would you prefer `Cpp format and lint`?

Reply:
It should be "C format and lint" since this game doesn't have any c++

Reply:
And yet the job name is cpp_lint :)
Done

Reply:
I think that means c preprocessor in this case, but yeah it's a weird name. thanks!